src/communication/azext_communication/version.py

```diff
@@ -4,7 +4,7 @@
 # --------------------------------------------------------------------------------------------
 
 
-VERSION = '1.6.0'
+VERSION = '2.0.0'
```

Comment:
Echoing earlier statements, we should see what the procedure is since we're depending on "preview" functionality